source/modules/src/main/sql/database-modules/aerius_general/02-functions/system.sql

```diff
@@ -0,0 +1,117 @@
+/*
+ * create_local_enum_and_casts
+ * ---------------------------
+ * Creates an enum in the current schema (first in search_path) by copying labels from a base enum (public.<enum_name>)
+ * and then adds implicit casts in both directions between the two enums.
+ */
+CREATE OR REPLACE FUNCTION system.create_local_enum_and_casts(v_enum_name text)
```

Comment:
Not sure why this is needed. Would assume that you'd import `aerius_general` as-is (no `import-into-schema`) once for a database, and then use `public.critical_deposition_area_type` where you reference the type in the other modules?